Hymns/Audio Player/AudioPlayerControlsView.swift

```diff
@@ -0,0 +1,63 @@
+import AVFoundation
+import SwiftUI
+
+struct AudioPlayerControlsView: View {
+
+    let player: AVPlayer
+    let timeObserver: PlayerTimeObserver
+    let itemObserver: PlayerItemObserver
+    @Binding var playbackState: PlaybackState
+    @State private var currentTime: TimeInterval = 0
+
+    var body: some View {
+        VStack {
+            if playbackState == .waitingForSelection {
+                Text("Buffering...")
+            }
+            Slider(value: $currentTime,
+                   in: 0...60,
+                   onEditingChanged: sliderEditingChanged,
+                   minimumValueLabel: Text("\(Utility.formatSecondsToHMS(currentTime))"),
+                   maximumValueLabel: Text("")) {
+                    // I have no idea in what scenario this View is shown...
+                    Text("seek/progress slider")
```

Comment:
The documentation says `- label: A `View` that describes the purpose of the instance.`

I'm guessing maybe this is for a11y? If that's the case, maybe you can put a more descriptive label

Reply:
Are you talking about the maximum value label? If so that is where the duration had gone before and where it will go again when we get duration working? It displays the current duration to the right of the bar. 

If you are talking about seek/progress slider I can change that?

Reply:
No, I'm talking about the ` // I have no idea in what scenario this View is shown...`